Source/WebKit/WebProcess/GPU/graphics/RemoteImageBufferProxy.cpp

```diff
@@ -359,21 +361,15 @@ std::unique_ptr<SerializedImageBuffer> RemoteImageBufferProxy::sinkIntoSerialize
 {
     ASSERT(hasOneRef());
 
-    flushDrawingContext();
```

Comment:
If image buffer flush is not happening, what happens to the unsent, unflushed commands? Why are they ok to discard?

Reply:
If you want to quickly test, you can try to change this to flushDrawingContextAsync() and see if it has any effect.

Reply:
I've already tested and `flushDrawingContextAsync` is measurably faster than `flushDrawingContext`, but it is also measurably than either to not flush at all.

My thinking was that the state of `m_needsFlush` can be passed to the new ImageBuffer once it's constructed.
...maybe there's something wrong with that?